### PR TITLE
I have added support for the DFRobot Romeo ESP32-S3 board.

### PR DESCRIPTION
**Pull request by fary99**
*By completing this PR sufficiently, you help us to review this Pull Request quicker and also help improve the quality of Release Notes*

### Checklist
1. [ ] Please provide specific title of the PR describing the change, including the component name (*eg. „Update of Documentation link on Readme.md“*)
2. [ ] Please provide related links (*eg. Issue which will be closed by this Pull Request*)
3. [ ] Please **update relevant Documentation** if applicable
4. [ ] Please check [Contributing guide](https://docs.espressif.com/projects/arduino-esp32/en/latest/contributing.html)

*This entire section above can be deleted if all items are checked.*

-----------
## Description of Change
Please describe your proposed Pull Request and it's impact.
I have added support for the DFRobot Romeo ESP32-S3 board.

## Tests scenarios
Please describe on what Hardware and Software combinations you have tested this Pull Request and how.

 I have tested my Pull Request on Arduino-esp32 core v2.0.9 withESP32-S3 Board with this scenario

## Related links
Please provide links to related issue, PRs etc.

(*eg. Closes #number of issue*)

### CODE REVIEW

File: variants/dfrobot_romeo_esp32s3/pins_arduino.h [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+static const uint8_t TX = 43;
+static const uint8_t RX = 44;
+
+static const uint8_t SDA = 1;
+static const uint8_t SCL = 2;
+
+
+static const uint8_t MOSI  = 15;
+static const uint8_t MISO  = 16;
+static const uint8_t SCK   = 17;
```
Comment (me-no-dev) [inline variants/dfrobot_romeo_esp32s3/pins_arduino.h:26]: You also need to add definition for `SS` here

Comment (fary99) [reply variants/dfrobot_romeo_esp32s3/pins_arduino.h:26]: Already added and committed

**Review by P-R-O-C-H-Y — APPROVED** [review]

**Comment by P-R-O-C-H-Y** [comment]
@fary99 The CI Board test fails with error:
```
  In file included from /home/runner/Arduino/hardware/espressif/esp32/cores/esp32/esp32-hal-gpio.h:29,
                   from /home/runner/Arduino/hardware/espressif/esp32/cores/esp32/esp32-hal.h:83,
                   from /home/runner/Arduino/hardware/espressif/esp32/cores/esp32/Arduino.h:36,
                   from /tmp/arduino/sketches/BBBCBC2F9559872EE54983F29332A0F0/sketch/CIBoardsTest.ino.cpp:1:
  /home/runner/Arduino/hardware/espressif/esp32/variants/dfrobot_romeo_esp32s3/pins_arduino.h:1: error: unterminated #ifndef
   #ifndef Pins_Arduino_h
   
  In file included from /home/runner/Arduino/hardware/espressif/esp32/cores/esp32/Arduino.h:222,
                   from /tmp/arduino/sketches/BBBCBC2F9559872EE54983F29332A0F0/sketch/CIBoardsTest.ino.cpp:1:
  /home/runner/Arduino/hardware/espressif/esp32/variants/dfrobot_romeo_esp32s3/pins_arduino.h:1: error: unterminated #ifndef
   #ifndef Pins_Arduino_h
   
  In file included from /home/runner/Arduino/hardware/espressif/esp32/libraries/SPI/src/SPI.h:25,
                   from /home/runner/work/arduino-esp32/arduino-esp32/libraries/ESP32/examples/CI/CIBoardsTest/CIBoardsTest.ino:2:
  /home/runner/Arduino/hardware/espressif/esp32/variants/dfrobot_romeo_esp32s3/pins_arduino.h:1: error: unterminated #ifndef
   #ifndef Pins_Arduino_h
```

You missed `#endif` for GDI_DISPLAY_FPC_INTERFACE part.

File: variants/dfrobot_romeo_esp32s3/pins_arduino.h [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+#define GDI_RES       38
+#define GDI_CS        18
+#define GDI_SDCS      0
+#define GDI_FCS       7
+#define GDI_TCS       12
+#define GDI_SCL       SCL
+#define GDI_SDA       SDA
+#define GDI_INT       13
+#define GDI_BUSY_TE   14
+
```
Comment (P-R-O-C-H-Y) [inline variants/dfrobot_romeo_esp32s3/pins_arduino.h:46]: Missing `#endif` for **GDI_DISPLAY_FPC_INTERFACE**

Comment (fary99) [reply variants/dfrobot_romeo_esp32s3/pins_arduino.h:46]: Ok, I've revised it

**Review by P-R-O-C-H-Y — APPROVED** [review]

**Pull request merged**